packages/vscode-extension/package.nls.json

```diff
@@ -373,14 +373,19 @@
     "teamstoolkit.taskDefinitions.args.prerequisites.nodejsTitle": "Validate if Node.js is installed.",
     "teamstoolkit.taskDefinitions.args.prerequisites.m365AccountTitle": "Sign-in prompt for Microsoft 365 account, then validate if the account enables the sideloading permission.",
     "teamstoolkit.taskDefinitions.args.prerequisites.portsTitle": "Validate available ports to ensure those debug ones are not occupied.",
-    "teamstoolkit.taskDefinitions.args.ports.title": "The port numbers to validate",
+    "teamstoolkit.taskDefinitions.args.portOccupancy.title": "The port numbers to validate",
     "teamstoolkit.taskDefinitions.args.ngrokArgs.title": "The arguments passed to the ngrok install command.",
     "teamstoolkit.taskDefinitions.args.ngrokPath.title":"The ngrok path.\nIf undefined, use Teams Toolkit ngrok, otherwise use provided ngrok.\ne.g.\"ngrokPath\": \"ngrok\" | \"C:/sample/path/ngrok\" | \"./bot/node_modules/ngrok/bin/ngrok\"",
     "teamstoolkit.taskDefinitions.args.tunnelInspection.title":"Tunnel inspection.\nIf undefined, get endpoint from log first, then the first PublicURL from default inspection \"http://127.0.0.1:4040/api/tunnels\"\nOtherwise, use provided inspection (the first PublicURL).",
     "teamstoolkit.taskDefinitions.args.env.title": "The environment name.",
     "teamstoolkit.taskDefinitions.args.output.title": "The keys of the environment variables of tunnel domain and tunnel endpoint.",
     "teamstoolkit.taskDefinitions.args.output.domain.title": "The key of the environment variable of tunnel domain.",
     "teamstoolkit.taskDefinitions.args.output.endpoint.title": "The key of the environment variable of tunnel endpoint.",
+    "teamstoolkit.taskDefinitions.args.output.id.title": "The key of the environment variable of dev tunnel id.",
```

Comment:
Can consider changing the 2nd ‘of’ to ‘for’

Reply:
Updated, thx!.